api/ruby/2fa_checker.rb

```diff
@@ -1,16 +1,33 @@
 require 'octokit.rb'
 
```

Comment:
The one thing that may help is to add some comments at the top of this file to explain some example command to running this. ie: `$ ruby 2fa_checker.rb githubtraining`to let the user of this script know what they need to pass in and what an expected result would be.